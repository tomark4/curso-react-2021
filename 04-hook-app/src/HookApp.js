// import CounterApp from './components/01-use-state/CounterApp';
// import CounterWithCustomHook from './components/01-use-state/CounterWithCustomHook';
// import SimpleForm from './components/02-use-effect/SimpleForm';
// import FormWithCustomHooks from './components/02-use-effect/FormWithCustomHooks';
// import MultipleCustomHooks from './components/03-examples/MultipleCustomHooks';
// import FocusScreen from "./components/04-useRef/FocusScreen";
// import RealExampleRef from "./components/04-use-ref/RealExampleRef";
// import LayoutEffect from './components/05-use-layout-effect/LayoutEffect'
// import MemoHook from './components/06-memos/MemoHook'
// import CallbackHook from './components/06-memos/CallbackHook'
// import Padre from "./components/07-tarea-memo/Padre";
// import TodoApp from './components/08-reducers/TodoApp'
import MainApp from './components/09-use-context/MainApp';

function HookApp() {
  return (
    <>
      {/* <CounterApp />

      <CounterWithCustomHook />

      <SimpleForm /> */}

      {/* <FormWithCustomHooks /> */}

      {/* <MultipleCustomHooks/> */}

      {/* <FocusScreen /> */}

      {/* <RealExampleRef /> */}

      {/* <LayoutEffect /> */}

      {/* <Padre /> */}

      <MainApp />
    </>
  );
}

export default HookApp;
