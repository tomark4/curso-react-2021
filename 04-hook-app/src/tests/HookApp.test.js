
import {shallow} from 'enzyme';
import HookApp from '../HookApp'

describe('Pruebas en hook app', () => {
  test('debe de retornar el componente', () => {
    let wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
})
