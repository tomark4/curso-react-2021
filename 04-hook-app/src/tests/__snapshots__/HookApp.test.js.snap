// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en hook app debe de retornar el componente 1`] = `
<Fragment>
  <MainApp />
</Fragment>
`;
